config/routes.rb

```diff
@@ -2,8 +2,7 @@
   namespace :api do
     namespace :v1 do
       get "/medical_professionals", to: 'medical_professionals#index'
-
-      post "/add_professional", to: 'add_professional#create'
+      post "/medical_professionals", to: 'medical_professionals#create'
```

Comment:
maybe make this singular since they're only adding one at a time.

Reply:
I've updated these to be resources instead of being hand-rolled. In REST though, changing pluralization establishes a new endpoint altogether.

For example: 
*'/medical_professionals'*
1. 'medical_professionals#index' will handle grabbing all of the doctor/mhp resources.
2. 'medical_professionals#show' would be for grabbing a single doctor/mhp resource.
3. 'medical_professionals#create' would be for creating a single doctor/mhp resource.
4. 'medical_professionals#update' would be for updating a single doctor/mhp resource.
5. 'medical_professionals#delete' would be for deleting a single doctor/mhp resource.

To my understanding, REST is the ability to change the verb of a single endpoint and continuously perform CRUD on a DB's resources. 

Let me know if that makes sense :)

Reply:
Ahhhh indeed. Yes, thank you. I blame morning brain 😵